charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.4.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.4.0
+version: 2.4.1
```

Comment:
As far as the chart is concerned this is new functionality so you should bump the minor.